src/contexts/search-context.ts

```diff
@@ -33,7 +31,6 @@ export function createSearchContext(
     sort: sortByArray.join(':') || '',
     indexUid,
     pagination: paginationState,
-    defaultFacetDistribution: defaultFacetDistribution || {},
```

Comment:
removed `initialFacetDistribution` from the `searchContext` as it is not correlated to the current search request but as value for the whole instance.